mockbee-test/ecom-store/src/App.js

```diff
@@ -1,13 +1,16 @@
 import "./App.css"
 import {Home} from "./pages/Home"
 import { Products } from "./pages/Products";
+import {Wishlist} from "./pages/Wishlist";
```

Comment:
have an index.js file in pages folder, import all pages over there and export them from there, so that your imports are clean in other components

Reply:
thanks, updated